### ISSUE TITLE
[feature]: add `inbound_fee_rate` as an option to `updatechanpolicy`

### ISSUE THREAD

**Issue by AndySchroder** [lightningnetwork/lnd#9165]
We have `fee_rate` and `fee_rate_ppm` in `updatechanpolicy` We also have `inbound_fee_rate_ppm` but we do not have `inbound_fee_rate`. Please add `inbound_fee_rate` for consistency.

**Comment by Roasbeef** [comment]
We have both fields, but perhaps the naming is unexpected -- `base_fee_msat` is the base fee: https://lightning.engineering/api-docs/api/lnd/lightning/update-channel-policy#lnrpcinboundfee

**Comment by hieblmi** [comment]
I think @AndySchroder means to have added the `lncli updatechanpolicy --inbound_fee_rate` cli option, which we have for the outbound fees.

**Comment by AndySchroder** [comment]
> I think @AndySchroder means to have added the `lncli updatechanpolicy --inbound_fee_rate` cli option, which we have for the outbound fees.

Yes, I'd like to be able to use the same unit multiplier for setting both inbound and outbound fees for both the base fee and the fee rate. It's very confusing to have to use different unit multipliers for inbound and outbound fees.

**Comment by litbot-9000** [comment]
Closing as duplicate of #8610 which tracks adding inbound fee configuration options. Unique details from this issue have been summarized there.
